Infrastructure/Componants/LeftSideMenu.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium;
+
+namespace Infrastructure
+{
+    public class LeftSideMenu : ComponantBase
+    {
+        private IList<IWebElement> Colors => ParentElement.FindElements(By.CssSelector("#ul_layered_id_attribute_group_3 .nomargin.hiddable.col-lg-6")).ToList();
```

Comment:
Not used, also why IList? If you are going to use it, maybe give it a more indicative name (ProductColors)?